gettsim/pensions.py

```diff
@@ -21,23 +18,23 @@ def pensions(person, tb, tb_pens):
 
     """
     # meanwages is only filled until 2016
-    yr = min(tb["yr"], 2016)
+    year = min(params["year"], 2016)
```

Comment:
Raise an ImplementationError if year > 2016 rather than silently doing something wrong.

Reply:
I can do that, but this will be a longer issue. I will need to adapt a lot of tests and etc.

Reply:
As I said, then just open an issue for it. This is orthogonal to the PR anyhow.

Reply:
Opened #114